src/docs/02-UX and Design Guidelines/colors.md

```diff
@@ -10,8 +10,8 @@ A colour palette comprises primary and accent colours that can be used for illus
       <dt class="frctl-palette-title">{{@key}}</dt>
       {{#each this}}
         <dd class="frctl-color">
-          <div class="frctl-color-swatch" style="background-color: {{@this}}">
-             <code class="frctl-color-name" data-bg-color="{{@this}}">{{@this}}</code>
+          <div class="frctl-color-swatch" style="background-color: {{@this}};">
+             <span class="frctl-color-name" data-bg-color="{{@this}}">{{@this}}</span>
```

Comment:
Why this change? It's showing a hex-code, which I think is "code"?

Reply:
The font is ugly. If I use code I need to overwrite a lot of styling. It is just not that important for me so changed it to span.

Reply:
I'm keeping the keys to the Dojo!